learn/cookbooks/digitalocean_droplet.md

```diff
@@ -1,129 +1,125 @@
-In any DigitalOcean page, when you are logged in, you will find a menu in the upper-right corner. Click on "Create" -> "Droplets".
+Once you log in to your DigitalOcean account, click the green "Create" button at the top-right of the page and select "Droplets".
 
-![Selecting "Droplets Create cloud servers" from the "Create" dropdown](/digitalocean/01.create.png)
+![Selecting "Droplets" from the "Create" dropdown](/digitalocean/create.png)
 
-### 2. Select Meilisearch snapshot
+### 2. Select a region for your Droplet
 
-By default, DigitalOcean will display the "Distributions" tab. Select the "Marketplace" tab and search for "meili". Select it.
+Select the region where you want to deploy your Droplet. Remember, the closer you are to your users, the better their search experience with Meilisearch will be.
 
-![Searching for 'meili' in Marketplace](/digitalocean/02.marketplace.png)
+![Selecting the London data center region](/digitalocean/select-region.png)
 
-### 3. Select your plan
+### 3. Select Meilisearch image
 
-Select your plan. Plans start at $5 (click on "See all plans" for more options). Memory-optimized options will give you better results for a production environment on big datasets.
+By default, DigitalOcean displays the "OS" tab. Select the "Marketplace" tab, search for "Meilisearch", and select the image.
 
-![Selecting the plan based on your usage](/digitalocean/03.select-plan.png)
+![Search results for 'Meilisearch' in Marketplace](/digitalocean/marketplace.png)
 
-### 4. Select a region for your droplet
+### 4. Choose Droplet size
 
-Select the region where you want to deploy your droplet. Remember, the closer you are to your users or customers, the better will be their search experience with Meilisearch.
+This is where you choose the amount of RAM, storage, and CPU cores your Droplet will have. Select your plan based on your needs. Memory-optimized options will give you better results when working with big datasets.
 
-![Selecting the London data center region](/digitalocean/04.select-region.png)
+![Selecting the plan based on your usage](/digitalocean/select-plan.png)
 
-### 5. Add your SSH key
+### 5. Choose an authentication method
 
-Select your SSH key in order to be able to connect to your droplet later. If you don't see your SSH key add yours to your account.
+You can either use SSH keys or a password to access your Droplet. We recommend using SSH keys as they are more secure.
 
-If you need help with this, visit [this link](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/to-account/)
+![Selecting SSH keys for authentication](/digitalocean/add-ssh-key.png)
 
-You can also set a password for `root` user if you prefer this authentication method, but it is less secure.
+Select the existing keys you want to add to your Droplet. If you don't have any existing keys, [follow DigitalOcean's instructions on how to create one](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/to-account/).
```

Comment:
Re-reading this made me think it was a bit weird to say "having/selecting existing keys"—you cannot select or have a non-existing key.

```suggestion
Select the SSH keys you want to add to your Droplet. If you don't have a key, [follow DigitalOcean's instructions on how to create one](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/to-account/).
```